Makefile

```diff
@@ -1,7 +1,9 @@
 DESTDIR ?= /
-moduledir ?= $(DESTDIR)/usr/share/mender/modules/v3
-inventorydir ?= $(DESTDIR)/usr/share/mender/inventory
-scriptdir ?= $(DESTDIR)/usr/lib/mender-configure/apply-device-config.d
+prefix ?= $(DESTDIR)
+moduledir ?= /usr/share/mender/modules/v3
+inventorydir ?= /usr/share/mender/inventory
+scriptdir ?= /usr/lib/mender-configure/apply-device-config.d
+systemd_unitdir ?= /lib/systemd
```

Comment:
Why don't we add `$(prefix)` in front of all of these instead, and save a lot of references below?

Also, is `prefix` actually necessary? I know it has historically been possible to use it, so I'm not against it, just trying to minimize things.

Reply:
@kacf two (possibly stupid) reasons for spreading out prefix:
- when I tried that, I ran into expansion problems and it did not get picked up correctly
- it is the same in the Mender client Makefile.
As for supporting the `prefix` directive itself, in my understanding this is one of the "canonical" parameters to Makefiles, so I usually expect it to be honoured.

Reply:
> when I tried that, I ran into expansion problems and it did not get picked up correctly

Hmm, ok. The pedantic in me wonders why that would cause expansion problems, but OTOH, I can't be bothered now, and this is fine too!

> As for supporting the `prefix` directive itself, in my understanding this is one of the "canonical" parameters to Makefiles, so I usually expect it to be honoured.

👍